ppr-ui/src/views/newMhrRegistration/HomeOwners.vue

```diff
@@ -179,23 +178,21 @@ export default defineComponent({
       isGlobalEditingMode,
       showGroups,
       getTotalOwnershipAllocationStatus,
-      hasMinimumGroups,
-      getInterestString
+      hasMinimumGroups
     } = useHomeOwners()
 
     const localState = reactive({
       showAddPersonSection: false,
       showAddPersonOrganizationSection: false,
-      disableAddHomeOwnerBtn: computed(() => {
-        return localState.showAddPersonOrganizationSection || localState.showAddPersonSection
-      }),
+      disableAddHomeOwnerBtn: computed(
+        () => localState.showAddPersonOrganizationSection || localState.showAddPersonSection
+      ),
```

Comment:
Not opposed to short form but would stick with whatever format is already in place here. 
If this is the norm for the component than all good 👍 

Just whatever is most consistent :)

Reply:
Sure thing, just thought I would clean up some redundant code...

Reply:
Totally! Looks good 👍